config/capsule.migrations/20161121212031_move_capsule_to_foreman_proxy_content.rb

```diff
@@ -0,0 +1,9 @@
+# migrate answers from capsule if exist
+answers['capsule'].each do |k|
+  if k.eql? 'capsule'
+    answers['foreman_proxy_content']['foreman_proxy_content'] = answers['capsule'].delete k
```

Comment:
Ah, okay, thanks. I was thinking there was a capsule option inside the class. Thanks!